### PR TITLE
Kp nucleus cyber nc protect

### PR DESCRIPTION
**Pull request by K-Patel-NC**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Review by v-admahe — CHANGES_REQUESTED** [review]

**Comment by v-admahe** [comment]
@K-Patel-NC : Please remove the logo and sample data file from the dataconnector folder.

Thanks,
Aditi Maheshwari

File: DataConnectors/NucleusCyber/NucleusCyberNCProtect.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{
+    "id": "NucleusCyberNCProtect",
+    "title": "Nucleus Cyber NC Protect",
+    "publisher": "Nucleus Cyber",
+    "descriptionMarkdown": "NC Protect connector provides the capability to ingest user activity logs and events into Azure Sentinel. The connector provides visibility into NCProtect user activity logs and events in Azure Sentinel to improve monitoring and investigation capabilities.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "NCProtectUAL",
```
[the remark this answers is not in the document]
Comment (K-Patel-NC) [reply DataConnectors/NucleusCyber/NucleusCyberNCProtect.json]: Hi @v-admahe, I have updated legend as per your request.

File: Sample Data/Custom/NCProtectUAL_CL.json [hunk 2]
```diff
@@ -0,0 +1,244 @@
+[
+	{
+		"TimeGenerated [UTC]": "3/21/2021, 10:38:57.794 PM",
+		"SHA512Hash_g": "",
```
Comment (v-admahe) [inline Sample Data/Custom/NCProtectUAL_CL.json:4]: As per the column name suffix (_g) its value should be guid. If value is empty for guid type then give it in this format "00000000-0000-0000-0000-000000000000".

Please update accordingly in all the field.

Thanks,
Aditi Maheshwari

**Review by v-admahe — CHANGES_REQUESTED** [review]

File: Sample Data/Custom/NCProtectUAL_CL.json (hunk 2 above)
Comment (v-admahe) [reply Sample Data/Custom/NCProtectUAL_CL.json:4]: We have update the sample data file and made the required changes in the file. Please find the sample data file below and  you can use this file for reference.
[NCProtectUAL.zip](https://github.com/Azure/Azure-Sentinel/files/6286312/NCProtectUAL.zip)

Thanks,
Aditi Maheshwari

Comment (K-Patel-NC) [reply Sample Data/Custom/NCProtectUAL_CL.json:4]: Hi @v-admahe Thanks for your update and sorry for delay.

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**